scripts/update_teams.py

```diff
@@ -85,13 +75,9 @@ def create_team(org, name, description, repos):
     if not os.path.exists(recipe):
         print("The {} feedstock is recipe less".format(package_name))
         continue
-    env = jinja2.Environment(undefined=NullUndefined)
-
-    with open(recipe) as fh:
-        contents = env.from_string(''.join(fh)).render(os=os, environ=os.environ)
-    data = yaml.safe_load(contents)
+    meta = MetaData(recipe)
```

Comment:
I think we just want the directory this is in because I believe this is the `meta.yaml`.